### ISSUE TITLE
sfpowerkit plugin is not Getting installed on Azure MS hosted agent

### ISSUE THREAD

**Issue by ayadav4** [Accenture/sfpowerkit#378]
HI Team,

In Azure devops Pipeline, I am using below command to install plugins MS hosted agent- hosted ubuntu 1604.

It was working till yesterday.

echo 'y' | npx sfdx plugins:install sfpowerkit.

error-

2020-09-02T04:18:34.2136289Z You acknowledge and agree that the CLI tool may collect usage information, user environment, and crash reports for the purposes of providing services or functions that are relevant to use of the CLI tool and product improvements. 
2020-09-02T04:18:34.2137155Z 
2020-09-02T04:18:34.9105889Z     TypeError: Cannot read property 'dim' of undefined

Regards,
Anil Yadav

**Comment by azlam-abdulsalam** [comment]
Hi Anil,
We have seen the reports of this error in particular ubuntu configuration, and it looks like a sfdx-cli one. The work around is to downgrade the CLI to 7.71.0, till it is fixed the cli

**Comment by gvivo** [comment]
Hi Team,

In GitLab CI/CD Pipelines we are currently facing a similar issue. This started happening this morning.
We attempt to install the plugin in a Kubernetes cluster with the following command:

echo 'y' | node_modules/sfdx-cli/bin/run plugins:install sfpowerkit

We receive the following error messages:

TypeError: Cannot read property 'dim' of undefined
ERROR: Job failed: exit code 1

We have tried the workaround to install the CLI 7.71.0 version, but we receive the same error.

Regards,
Guillem Vivó

**Comment by azlam-abdulsalam** [comment]
Here is another workaround listed in Stack overflow 
https://salesforce.stackexchange.com/questions/318537/sfdx-plugin-installation-crash-typeerror-cannot-read-property-dim-of-undefin

**Comment by rody** [comment]
Hello,

I was able to reproduce this issue yesterday in a Docker container. The issue comes from a change in sfdx-cli (or one of its dependencies). This issue affects the installation of many plugins (not only sfpowerkit)

## Known working versions:
 - sfdx-cli@7.70.1 (and earlier)

## Known broken versions
 - sfdx-cli@7.71.0
 - sfdx-cli@7.72.0

## Workaround fix:
Specify the sfdx-cli version when installing with npm or yarn:
`npm install -g sfdx-cli@7.70.1`

## Possible source of the bug
The `dim` function seems to be part of the [chalk](https://github.com/chalk/chalk) library. I suspect that the issue appears when the cli is writing data to the console. I'll have to investigate further to confirm this.

Note: I've tried multiple combinations of debian/ubuntu and node and the results are the same.

**Comment by ayadav4** [comment]
Hi Azlam,

I Installed sfdx cli version 7.70.1. It is working for me.

Regards,
Anil Yadav

**Comment by gvivo** [comment]
Hi Team,

Thanks for the quick answers. 
We followed the workaround of installing the sfdx cli version 7.70.1 and it's also working here.

Regards,
Guillem Vivó

**Comment by rody** [comment]
For info, this issue seems to be present also in the 7.73.1 release
I tested with:

```docker
FROM node:14-stretch
RUN npm install -g sfdx-cli@7.73.1
RUN echo 'y' | sfdx plugins:install sfpowerkit
CMD "bash"
```

**Comment by rody** [comment]
I can confirmed it was fixed in 7.74.1
see https://github.com/forcedotcom/cli/issues/592

**Comment by azlam-abdulsalam** [comment]
Closing this as its fixed in 7.74.1
